obfuscator-core/src/main/java/me/superblaubeere27/jobf/ui/GUI.java

```diff
@@ -400,7 +401,12 @@ private void startObfuscator() {
                     JObfImpl.INSTANCE.processJar(config);
                 } catch (Throwable e) {
                     e.printStackTrace();
-                    JOptionPane.showMessageDialog(this, e.toString(), "ERROR", JOptionPane.ERROR_MESSAGE);
+	
+                    String msg = e.toString() + "\n";
+                    
+					msg += Throwables.getStackTraceAsString(e);
+					
+                    JOptionPane.showMessageDialog(this, msg, "ERROR encountered at " + e.getStackTrace()[0].getClassName(), JOptionPane.ERROR_MESSAGE);
```

Comment:
I think this would look a way better if it was in a text area... We can change it later ^^

Reply:
Good idea, I will write that now